src/datanode/src/instance/grpc.rs

```diff
@@ -364,15 +366,44 @@ mod test {
                 schema_name: "my_database".to_string(),
                 table_name: "my_table".to_string(),
                 kind: Some(alter_expr::Kind::AddColumns(AddColumns {
-                    add_columns: vec![AddColumn {
-                        column_def: Some(ColumnDef {
-                            name: "b".to_string(),
-                            datatype: ColumnDataType::Int32 as i32,
-                            is_nullable: true,
-                            default_constraint: vec![],
-                        }),
-                        is_key: true,
-                    }],
+                    add_columns: vec![
+                        AddColumn {
+                            column_def: Some(ColumnDef {
+                                name: "b".to_string(),
+                                datatype: ColumnDataType::Int32 as i32,
+                                is_nullable: true,
+                                default_constraint: vec![],
+                            }),
+                            is_key: true,
+                            location: None,
+                        },
+                        AddColumn {
+                            column_def: Some(ColumnDef {
+                                name: "c".to_string(),
+                                datatype: ColumnDataType::Int32 as i32,
+                                is_nullable: true,
+                                default_constraint: vec![],
+                            }),
+                            is_key: true,
+                            location: Some(Location {
+                                location_type: LocationType::First.into(),
+                                after_cloumn_name: "".to_string(),
+                            }),
+                        },
+                        AddColumn {
+                            column_def: Some(ColumnDef {
+                                name: "d".to_string(),
+                                datatype: ColumnDataType::Int32 as i32,
+                                is_nullable: true,
+                                default_constraint: vec![],
+                            }),
+                            is_key: true,
+                            location: Some(Location {
+                                location_type: LocationType::After.into(),
+                                after_cloumn_name: "a".to_string(),
+                            }),
+                        },
+                    ],
```

Comment:
Please assert the query result of `select *` to ensure that the order of the columns is correct and columns are added. If we don't assert, we don't know if it works.